### PR TITLE
tea: new package at 0.4.1

### PR DESCRIPTION
**Pull request by mmatongo**
## Description of package
A command line tool to interact with Gitea servers

## Installed manifest of package

```
/var/db/kiss/installed/tea/version
/var/db/kiss/installed/tea/sources
/var/db/kiss/installed/tea/manifest
/var/db/kiss/installed/tea/files/bash_autocomplete
/var/db/kiss/installed/tea/files/
/var/db/kiss/installed/tea/depends
/var/db/kiss/installed/tea/checksums
/var/db/kiss/installed/tea/build
/var/db/kiss/installed/tea/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/bash-completion/completions/tea
/usr/share/bash-completion/completions/
/usr/share/bash-completion/
/usr/share/
/usr/bin/tea
/usr/bin/
/usr/
```

## New package

- [Y] Latest upstream version.
- [Y] I agree to maintain this package.
- [Y] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

File: community/tea/files/bash_autocomplete [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    else
+      opts=$( ${COMP_WORDS[@]:0:$COMP_CWORD} --generate-bash-completion )
+    fi
+    COMPREPLY=( $(compgen -W "${opts}" -- ${cur}) )
+    return 0
+  fi
+}
+
+complete -o bashdefault -o default -o nospace -F _cli_bash_autocomplete $PROG
+unset PROG
```
Comment (dylanaraps) [inline community/tea/files/bash_autocomplete:21]: I don't think we should ship completion files which packages. Can this not be sent upstream?

Comment (mmatongo) [reply community/tea/files/bash_autocomplete:21]: Yes it can, I will push a new commit.

File: community/tea/build [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -e
+
+make build
+
+install -Dm755 tea "$1/usr/bin/tea"
+install -Dm644 bash_autocomplete "$1/usr/share/bash-completion/completions/tea"
```
Comment (dylanaraps) [inline community/tea/build:6]: This needs to also be removed.

Comment (mmatongo) [reply community/tea/build]: Forgot to remove that

**Pull request merged**